docs/root/configuration/http/http_filters/cache_filter.rst

```diff
@@ -0,0 +1,46 @@
+.. _config_http_filters_cache:
+
+HTTP Cache Filter
+=================
+**Work in Progress--not ready for deployment**
+
+HTTP caching can improve system throughput and latency and reduce network and
+backend loads when the same content is requested multiple times. Caching is
+particularly valuable for edge proxies and browser-based traffic, which
+typically include many cacheable static resources, but it can be useful any time
+there is enough repeatedly served content.
+
+Configuration
+-------------
+* :ref:`Configuration API <envoy_api_msg_config.filter.http.cache.v2.cacheConfig>`
+* This filter should be configured with the name *envoy.filters.http.cache*.
+
+The only required configuration field is :ref:`name
+<envoy_api_field_config.filter.http.cache.v2.CacheConfig.name>`, which must
+specify a valid cache storage implementation linked into your Envoy
+binary. Specifying 'SimpleHttpCache' will select a proof-of-concept
```

Comment:
Can you use reverse DNS naming (with `envoy.simple_http_cache` for builtins) as with all other extension naming in Envoy?

Reply:
Actually, maybe something like `envoy.extensions.cache.http.simple` (or however we want this to hang in the API `envoy.extensions` tree and in source in `source/extensions`).

Reply:
Changed to envoy.extensions.http.cache.simple (http before cache), which I think is what you intended.